src/test/java/com/mongodb/hibernate/jdbc/MongoClientCustomizerTests.java

```diff
@@ -84,6 +88,11 @@ void testMongoClientCustomizerTakeEffect(boolean customizerAppliesConnectionStri
 
             var clusterDescription = mongoClient.getCluster().getClusterId().getDescription();
             assertEquals(applicationName, clusterDescription);
+
+            var driverInformation = mongoClient.getMongoDriverInformation();
+            assertAll(
+                    () -> assertTrue(driverInformation.getDriverNames().contains(BuildConfig.NAME)),
```

Comment:
the  driver names will contain 'sync' into the bargin.

Reply:
Yeah, the `MongoClients` method adds that, though without a version